fe/fe-core/src/main/java/org/apache/doris/qe/MysqlConnectProcessor.java

```diff
@@ -195,6 +198,58 @@ private void handleExecute() {
         handleExecute(preparedStatementContext.command, stmtId, preparedStatementContext);
     }
 
+    private String parseRealSql(String origStmt, Map<PlaceholderId, Expression> idExpressionMap) {
+        if (idExpressionMap.isEmpty()) {
```

Comment:
I think it's tricky to rewrite the sql, and i think maybe another way, we could print EXETUE xxx USING XXX , and add another filed  in audit log to print the prepared sql like `select * from tbl where a = ?`

Reply:
Modified